FRENCH/src/translation-terms.md

```diff
@@ -15,6 +15,7 @@ français.
 | associated function | fonction associée | - |
 | binary crate | crate binaire | s'utilise au féminin |
 | *n*-bit number | nombre encodé sur *n* bits | - |
+| boilerplate code | code standard | - |

```

Comment:
May not need a particular fixed translation, and should be reajusted depending on the context instead

Reply:
Of course !